sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuDeviceManager.scala

```diff
@@ -315,9 +315,27 @@ object GpuDeviceManager extends Logging {
       }
 
       Cuda.setDevice(gpuId)
-      Rmm.initialize(init, logConf, poolAllocation)
-      RapidsBufferCatalog.init(conf)
+      try {
+        Rmm.initialize(init, logConf, poolAllocation)
+      } catch {
+        case firstEx: CudfException if ((init & RmmAllocationMode.CUDA_ASYNC) != 0) => {
+          logWarning("Failed to initialize RMM with ASYNC allocator. " +
+            "Try to initialize it again with ARENA allocator as a fallback option.")
```

Comment:
This should say: `Initializing with ARENA allocator as a fallback option." otherwise it feels we are giving the user a suggestion that nothing worked, they should stop spark, and start it again with ARENA.